voorhoede/head-start
# chore: generator scripts

# Changes

Each new Block, Component, API or Page route requires a few files with some default plumbing. This PR adds generator scripts to scaffold these new Block, Component, API & page route files. This should help the consistency in the project and save developers some time.

# Associated issue

N/A

# How to test

1. Checkout this branch
2. Run `npm run create` (or `npm run create:api | block | component | page`)
3. Follow the prompts
4. Check the generated files

# Checklist

- ~~I have added/updated tests to prove that my feature works (if not possible please explain why)~~
- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
